empty-settings.gradle

```diff
@@ -0,0 +1,2 @@
+// Empty settings file to ignore subprojects
```

Comment:
This file seems to be unrelated to the PR. Can you remove it for now?

Reply:
Actually, see line 9 of `.gitlab-ci.yml ` -- it solves the problem of the built-in Gradle not being able to process all the subproject of our `build.gradle`.

Reply:
@schildbach Do yo see a better way of solving this issue?

Reply:
Somehow I don't see the problem. For example see https://gitlab.com/bitcoinj/bitcoinj/-/jobs/260248387, it has run all the subprojects despite me having removed the empty-settings.gradle. Or am I missing something?

Reply:
OK, well that's cool. I probably changed something else that made the issue go away. It's likely that I had some other changes to the build files that weren't compatible with the gradle installed with apt-get. We may run into that in the future. The problem is that running `gradle wrapper` on a project requires the current version of gradle to be able to successfully process the root `build.gradle` and all the subprojects.

I'll go head and remove it.